.circleci/config.yml

```diff
@@ -908,7 +926,10 @@ workflows:
           requires: [build]
           filters: *all_branches_and_tags
       - notify_services:
```

Comment:
Actually shall this be from helm repo???

Reply:
https://github.com/omgnetwork/helm-development/issues/478

Reply:
yep i think that would be a better place to do it, just because the increase_chart_version_XXX has been run doesn't mean it's deployed

Reply:
I think it will have to be separate PR as I don't even know how to do that yet 😛